autoPyTorch/pipeline/components/training/trainer/base_trainer_choice.py

```diff
@@ -98,11 +98,11 @@ def get_components(self) -> Dict[str, autoPyTorchComponent]:
         return components
 
     def get_hyperparameter_search_space(
-        self,
-        dataset_properties: Optional[Dict[str, str]] = None,
-        default: Optional[str] = None,
-        include: Optional[List[str]] = None,
-        exclude: Optional[List[str]] = None,
+            self,
```

Comment:
this is over indented, through the project we are following a 4 space tab for continuation indents

Reply:
It also happened somehow, but I will get them back, thanks.